common/src/View/FileTextureCollectionEditor.cpp

```diff
@@ -174,7 +175,7 @@ namespace TrenchBroom {
             auto collections = document->enabledTextureCollections();
 
             const auto index = static_cast<size_t>(m_collections->currentRow());
-            VectorUtils::swapPred(collections, index);
```

Comment:
needs an ensure(index > 0, "???");

Reply:
I have opted to add an if here. I used to think it's better to catch the errors and fix the cause, but in some cases it's just better to be defensive IMO.